tests/unit/test_rollouts.py

```diff
@@ -39,7 +39,11 @@ def test_buckets(self):
             # should be [0..33], [34..66], and [67..100]. Anything from [101..200]
```

Comment:
nit: I think this comment needs to be updated, right?

Reply:
ah good catch